### PR TITLE
Need to set VolumeContentSource if creating volume from snapshot

### LINKED ISSUE #50: Need to set VolumeContentSource if creating volume from snapshot
In CreateVolume, we should set VolumeContentSource after creating volume from snapshot is successful.

	return &csi.CreateVolumeResponse{
		Volume: &csi.Volume{
			VolumeId:      volumeID,
			CapacityBytes: req.GetCapacityRange().GetRequiredBytes(),
			VolumeContext: req.GetParameters(),
		},

### PR DESCRIPTION
**Pull request by zhucan**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind bug

**What this PR does / why we need it**:
Need to set VolumeContentSource if creating volume from snapshot

**Which issue(s) this PR fixes**:

Fixes #50

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
Set volume content source if creating volume from snapshot.
```

### CODE REVIEW

**Comment by xing-yang** [comment]
Please add a release note:

```release-note
Set volume content source if creating volume from snapshot.
```

File: pkg/hostpath/controllerserver.go [hunk 1]
```diff
@@ -193,6 +193,17 @@ func (cs *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVol
 		}
 	}
 
+	if req.GetVolumeContentSource() != nil {
+		return &csi.CreateVolumeResponse{
```
Comment (xing-yang) [inline pkg/hostpath/controllerserver.go]: This and the return on line 207 can be combined into one return.  Set ContentSource in CreateVolumeResponse first if needed, and then return the response.

Comment (zhucan) [reply pkg/hostpath/controllerserver.go]: copy that

**Comment by zhucan** [comment]
> Please add a release note:
> 
> ```
> Set volume content source if creating volume from snapshot.
> ```

copy that

**Comment by xing-yang** [comment]
Code looks good.  Please fix the description section:

What type of PR is this?
/kind bug

**Comment by zhucan** [comment]
> Code looks good. Please fix the description section:
> 
> What type of PR is this?
> /kind bug

c

**Comment by zhucan** [comment]
> Code looks good. Please fix the description section:
> 
> What type of PR is this?
> /kind bug

copy that

**Pull request merged**